WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -83,7 +83,7 @@ public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo in
 					var pockets = wallet.GetPockets();
 					var spentCoins = transaction.SpentCoins;
 					var usedPockets = pockets.Where(x => x.Coins.Any(coin => spentCoins.Contains(coin)));
-					var coinsToUse = usedPockets.SelectMany(x => x.Coins).ToImmutableArray();
+					var coinsToUse = usedPockets.SelectMany(x => x.Coins).Where(coin => coin.IsAvailable()).ToImmutableArray();
```

Comment:
Is there a reason why not doing `wallet.GetPockets(onlyAvailableCoins: true);` in L83 instead of this?

Reply:
No, I don't think so.